src/main/java/tagalong/subsystems/micro/Pivot.java

```diff
@@ -283,7 +293,9 @@ public double motorToPivotRot(double motorRot) {
     if (_isMicrosystemDisabled) {
       return 0.0;
     }
-    // FUTURE DEV: modify to allow for unfused or not 1:1 with pivot
+    if (_pivotType == PivotTypes.noEncoder) {
+      return motorRot / _motorToMechRatio;
+    }
     return motorRot / (_encoderToPivotRatio * _motorToEncoderRatio);
```

Comment:
this is not accounting for fused currently--for fused, you just want to return motorRot without extra conversion